install-dev/langs/mx/language.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<language>
+	<name><![CDATA[Español MX (Spanish)]]></name>
+	<locale>es-MX</locale>
+	<language_code>es</language_code>
```

Comment:
Should it be _es_ or _es_mx_? cf. https://github.com/PrestaShop/PrestaShop/blob/develop/app/Resources/all_languages.json

Reply:
No, es-MX is fine :)